### ISSUE TITLE
[Feature]: ignore global_step?

### ISSUE THREAD

**Issue by Freed-Wu** [wandb/wandb#4448]
### Description

```python
#!/usr/bin/env python
"""main."""
import wandb
from torch.utils.tensorboard.writer import SummaryWriter

model = image_models["mbt2018"](pretrained=True, quality=6)

log_dir = '/output/logs'

name = '5'
wandb.init(project="my-test-project-2", dir=log_dir, name=name, id=name,
           config={"learning_rate": 0.001, "epochs": 100, "batch_size": 128},
           sync_tensorboard=True)

writer = SummaryWriter(log_dir=log_dir)

writer.add_scalars('metrics/loss', {'train': 20, 'val': 100}, 0)
writer.add_scalars('metrics/loss', {'train': 10, 'val':  50}, 1)

# Optional
wandb.finish()
```

```
wandb: Run history:
wandb:                          global_step ▁█
wandb:  logs/metrics_loss_train/global_step ▁█
wandb: logs/metrics_loss_train/metrics/loss █▁
wandb:    logs/metrics_loss_val/global_step ▁█
wandb:   logs/metrics_loss_val/metrics/loss █▁
wandb:
wandb: Run summary:
wandb:                          global_step 1
wandb:  logs/metrics_loss_train/global_step 1
wandb: logs/metrics_loss_train/metrics/loss 10.0
wandb:    logs/metrics_loss_val/global_step 1
wandb:   logs/metrics_loss_val/metrics/loss 50.0
```

Can it ignore uploading any global_step?

### Suggested Solution

```python
wandb.init(sync_tensorboard=True, ignore='.*global_step')
```

**Comment by ArtsiomWB** [comment]
Hi Freed-Wu! 
Would you mind explaining your use case a little bit, and why you would like to ignore the global_step?

**Comment by Freed-Wu** [comment]
Because global_step generated by tensorboard is always a line in wandb and will not occur in tensorboard:

This is wandb:

![screen-2022-11-04-22-44-28](https://user-images.githubusercontent.com/32936898/200003372-763ad8bf-0cf8-457a-baa9-ab46b3f246da.jpg)

This is tensorboard:

![screen-2022-11-04-22-44-06](https://user-images.githubusercontent.com/32936898/200003355-37555673-8831-4938-8c53-2db58679cfba.jpg)

**Comment by Freed-Wu** [comment]
My meaning is tensorboard will not display a graph which x axis is step and y axis is global step (that is y=x). but wandb will display this graph. I think if we sync_tensorboard=True, wandb should keep confirmity with tensorboard.

**Comment by ArtsiomWB** [comment]
Hi @Freed-Wu! Currently, we do not have a way to prevent global-step from showing in your dashboard, but what you can do is remove this chart manually by pressing the three dots icon in the top right corner of the graph, and pressing the delete panel option.

Tensorboard is an integration with wandb rather than being a part of it, so the wandb charts will be uploaded into your tensorboard tab.

If you would like to I could submit a feature request ticket to our engineers, so we can see this added in the future.

**Comment by Freed-Wu** [comment]
> Tensorboard is an integration with wandb rather than being a part of it, so the wandb charts will be uploaded into your tensorboard tab.

Oh, it is right.

> If you would like to I could submit a feature request ticket to our engineers, so we can see this added in the future.

Great. I think most people who changes from tensorboard to wandb need this function :+1:

**Comment by ArtsiomWB** [comment]
Hi @Freed-Wu! Your feature request has been submitted to our engineers.
Cheers,
Have a great weekend :)
